staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -423,7 +424,7 @@ func (c *Config) AddPostStartHook(name string, hook PostStartHookFunc) error {
 		return fmt.Errorf("hook func may not be nil: %q", name)
 	}
 	if c.DisabledPostStartHooks.Has(name) {
-		klog.V(1).Infof("skipping %q because it was explicitly disabled", name)
+		klog.V(1).InfoS("Skipping postStartHook because it was explicitly disabled", "hookName", name)
```

Comment:
s/postStartHook/PostStartHook/

Reply:
Applied and sorry for not unified  style. Thanks